packages/engine/src/simulation/package/init/packages/js_py/package.js

```diff
@@ -82,18 +82,23 @@ const run_task = (experiment, _sim, task_message, _group_state, context) => {
   try {
     agents = init_fn(context);
   } catch (e) {
-    Error.prepareStackTrace = prepare_user_trace;
-    const trace = e.stack;
-    throw new Error(JSON.stringify(trace));
-  }
+    // TODO
+        // Error.prepareStackTrace = prepare_user_trace;
+        const trace = e.stack;
+        throw new Error(JSON.stringify(trace));
+    }
 
   if (!Array.isArray(agents)) {
     throw new Error(`init must return an array not '${typeof agents}'`);
   }
 
   let data;
   try {
-    data = JSON.stringify(agents);
+    data = {
+            "SuccessMessage": {
+                "agents": agents
+            }
+        };
   } catch (e) {
     throw new Error(
       `could not serialize init return value to JSON: ${e.message}`,
```

Comment:
This error message is incorrect now.

Reply:
I'm not sure it's incorrect, just shouldn't be able to happen. Will update and remove